packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -645,11 +644,28 @@ async function downloadDocsAndUpsertFHIR({
             processFhirAndSearchResponse(patient, doc, fhir);
           }
 
-          completedCount++;
+          await tallyDocQueryProgressWithSource({
+            patient: { id: patient.id, cxId: patient.cxId },
+            progress: {
+              successful: 1,
+            },
+            type: "download",
+            requestId,
+            source: MedicalDataSource.COMMONWELL,
+          });
 
           return FHIRDocRef;
         } catch (error) {
-          errorCount++;
+          await tallyDocQueryProgressWithSource({
+            patient: { id: patient.id, cxId: patient.cxId },
+            progress: {
+              errors: 1,
+            },
+            type: "download",
+            requestId,
+            source: MedicalDataSource.COMMONWELL,
```

Comment:
OOC, why don't we do something like this explicitly when querying carequality as well?

Reply:
Also, if conversion also happens right above in the call to `convertCDAToFHIR`, then why dont we also call `tallyDocQueryProgressWithSource` with type "conversion"

Reply:
> OOC, why don't we do something like this explicitly when querying carequality as well?
Its in [this PR](https://github.com/metriport/metriport/pull/1437)

Reply:
> Also, if conversion also happens right above in the call to `convertCDAToFHIR`, then why dont we also call `tallyDocQueryProgressWithSource` with type "conversion"

We tally the conversion after its been done by the lambda that way we can tally if it was successful or errored